### PR TITLE
Update channels

### PR DESCRIPTION
**Pull request by davidclaeysquinones**
Adds additional channels necessary for PR https://github.com/iptv-org/epg/pull/2485

### CODE REVIEW

**Review by freearhey — CHANGES_REQUESTED** [review]
- extra space before the channel name and logo url
- `alt_names` should be specified only if it differs from the official name
- logo size should not exceed 512x512 pixels

https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#data-scheme

**Comment by davidclaeysquinones** [comment]
I've removed unnecessary alt-names and spaces.
Furthermore I've resized the image that exceeded the permitted resolution. 
The Runtime channels have `1920x1080` in the file name but their resolution is actually 360x180.

File: data/channels.csv [hunk 1]
```diff
@@ -28867,6 +28878,7 @@ Televisiete.gt,Televisiete,,,,GT,,,c/GT,spa,,FALSE,,,,https://www.chapintv.com/,
 TelevisionA1.do,Television A1,Televisión A1,,,DO,,,c/DO,spa,,FALSE,,,,https://www.televisiona1.com/,https://www.televisiona1.com/images/logo.png
 TelevisionA1Canal2.do,Television A1 Canal 2,Televisión A1 Canal 2,,,DO,,,c/DO,spa,,FALSE,,,,https://www.televisiona1.com/canal-2,https://www.televisiona1.com/images/logo.png
 TelevisionAranda.es,Television Aranda,Televisión Aranda,,,ES,ES-CM,,s/ES-CM,spa,,FALSE,,,,http://www.telearanda.es/,https://i.imgur.com/FwIIdKG.png
+TelevisioCastello.es,Televisio Castello, Televisió de Castelló,,,ES,ES-VC,,s/ES-VC,cat,,FALSE,,,,https://www.tvcs.tv/,https://imgur.com/a/FF8klN9
```
Comment (freearhey) [inline data/channels.csv:28881]:
```suggestion
TelevisioCastello.es,Televisio Castello, Televisió de Castelló,,,ES,ES-VC,,s/ES-VC,cat,,FALSE,,,,https://www.tvcs.tv/,https://i.imgur.com/SbgQbC8.jpeg
```

File: data/channels.csv [hunk 2]
```diff
@@ -20553,6 +20555,7 @@ NauticalChannel.it,Nautical Channel,,,,IT,,Rome,c/IT,eng;ita;fra,sports,FALSE,,,
 NAUTVNorthernArizonaUniversity.us,NAU-TV Northern Arizona University,,,Northern Arizona University,US,US-AZ,Flagstaff,c/US,eng,education,FALSE,,,,,https://i.imgur.com/SHiYGII.png
 NavahangTV.fi,Navahang TV,,,,FI,,Helsinki,c/FI,fas,music,FALSE,,,,https://www.navahang.com/,https://i.imgur.com/YteKXJy.png
 NavarraTV.es,Navarra TV,,,,ES,,Pamplona,c/ES,spa,general,FALSE,,,,https://www.navarratelevision.es/,https://i.imgur.com/98MvIRA.png
+NavarraTV2.es,Navarra TV 2,,,,ES,,Pamplona,c/ES,spa,general,FALSE,,,,https://www.navarratelevision.es/,https://es.wikipedia.org/wiki/Archivo:Logonavarratv2.jpg
```
Comment (freearhey) [inline data/channels.csv:20558]:
```suggestion
NavarraTV2.es,Navarra TV 2,,,,ES,,Pamplona,c/ES,spa,general,FALSE,,,,https://www.navarratelevision.es/,https://upload.wikimedia.org/wikipedia/commons/f/f9/Logonavarratv2.jpg
```

**Review by freearhey — CHANGES_REQUESTED** [review]

**Review by freearhey — APPROVED** [review]

**Review by Alstruit — APPROVED** [review]

**Pull request merged**